server/models/base_persister.go

```diff
@@ -0,0 +1,146 @@
+package models
+
+import (
+	"strings"
+
+	"github.com/gofrs/uuid"
+	"github.com/meshery/meshkit/database"
+	"gorm.io/gorm"
+)
+
+// BasePersister provides common database operations for all persisters
+type BasePersister struct {
```

Comment:
```suggestion
type BaseDBPersister struct {
```

Since it is only db related I think we should add it in the name but not sure, what do you think about it ?